src/main/webapp/i18n/de/result.json

```
@@ -42,10 +42,10 @@
             "manualResult": "Manuelles Ergebnis",
             "invalidScore": "Eine Bewertung ist erforderlich und muss eine Zahl zwischen 0 und 100 sein.",
             "resultStringRequired": "Ein Ergebnistext ist erforderlich.",
-            "studentLogin": "Studenten-Login",
-            "studentLoginRequired": "Studenten-Login ist erforderlich.",
+            "studentLogin": "Studierenden-Login",
+            "studentLoginRequired": "Studierenden-Login ist erforderlich.",
             "exerciseStillOpen": "Die Aufgabe ist noch für Einreichungen geöffnet!",
-            "studentOverride": "Die Einreichungsfrist dieser Aufgabe ist noch nicht erreicht. Der Student könnte das manuelle Ergebnis noch mit einer neuen Lösung überschreiben.",
+            "studentOverride": "Die Einreichungsfrist dieser Aufgabe ist noch nicht erreicht. Studierende könnten das manuelle Ergebnis noch mit einer neuen Lösung überschreiben.",
```

```suggestion
            "studentOverride": "Die Einreichungsfrist dieser Aufgabe ist noch nicht erreicht. Studierende können das manuelle Ergebnis noch mit einer neuen Lösung überschreiben.",
```
The subjunctive seems superfluous